Minor content updates round1 (#3365)
twilio-labs/paste: packages/paste-website/src/pages/components/list/index.mdx

Lists are a number of connected list items that are printed consecutively, typically one after the other. Lists can be ordered, using numbers as an indicator to the reader, or unordered with bullet points.

- Ordered lists are displayed with a preceding number.
- Unordered lists are displayed with a preceding bullet point.
- Lists can be nested as deeply as needed, and they may alternate between ordered and unordered.
- Lists should align with the font-size, font-weight, and line-height of paragraphs, but are styled with bullets or numbers.
- Lists can have similar children elements as paragraphs to provide emphasis on a certain word or phrase.

### Accessibility

> - List text should meet AA requirements (4.5:1) for color contrast from itself and the background color.

Reviewer: ```suggestion
```

We don't mention this requirement for all our other text components

- `ListItem`s should always have an `OrderedList` or `UnorderedList` parent component.

> - Nested `ListItem`s should be wrapped in their own, new `OrderedList` or `UnorderedList` parent component, e.g.

Reviewer: ```suggestion
- Nested `ListItem`s should be wrapped in their own, new `OrderedList` or `UnorderedList` parent component. For example:
```

</div>`}
</LivePreview>

## Composition notes

Start each list item with the same part of speech, usually a noun or verb. This makes the list "parallel."

Do not use a period at the end of list items unless they are complete sentences. Do not use commas or ellipses at the end of list items.

> Avoid listing more than very long lists. Where possible, make a big list into smaller ones with explanatory [Headings](/components/heading).

Reviewer: ```suggestion
Avoid listing very long lists. Where possible, make a big list into smaller ones with explanatory [Headings](/components/heading).
```
